feature-libs/checkout/base/components/checkout-payment-method/checkout-payment-form/checkout-payment-form.module.ts

```diff
@@ -31,6 +33,9 @@ import { CheckoutBillingAddressFormModule } from '../../checkout-billing-address
     FormErrorsModule,
     FeaturesConfigModule,
     CheckoutBillingAddressFormModule,
+    FormRequiredAsterisksComponent,
+
```

Comment:
This is just a nitpick but since its appearing in many modules, lets remove this empty line ;)